string loc_name
---
bool is_possible_go

